spatialos-sdk/src/worker/core/parameters.rs

```diff
@@ -86,11 +100,29 @@ impl ConnectionParameters {
     }
 }
 
+pub enum ProtocolType {
+    Tcp(TcpNetworkParameters),
+    RakNet(RakNetNetworkParameters),
+}
+
+impl ProtocolType {
+    fn to_worker_sdk(&self) -> (u8, Option<Worker_RakNetNetworkParameters>, Option<Worker_TcpNetworkParameters>) {
```

Comment:
Not sure if there's a nicer way of implementing this. Based on its usage these don't have to be options and should just return default values.

Reply:
Swapped the impl to return `(u8, Worker_RakNetNetworkParameters, Worker_TcpNetworkParameters)`